### ISSUE TITLE
Suggestion: option to see video view date

### ISSUE THREAD

**Issue by fireton** [sniklaus/youtube-watchmarker#69]
It could be very useful if where was option to see a date when I watched the video. For instance instead of simple «WATCHED» it would say «WATCHED 20.08.2021».

**Comment by shoaibshakeel381** [comment]
What if you watch a video multiple times? Which time do you expect to see? 

Btw adding more information will increase storage size. I've been using this extensions for many years, now extension causes huge cpu spike right after a new video is opened. So I would suggest not to added extra data if possible or atleast have an option to toggle it.

**Comment by fireton** [comment]
@shoaibshakeel381 Last viewed time, obviously.  Date viewed stored anyway, you can see it in the list of all viewed videos on settings page.

**Comment by sniklaus** [comment]
Thanks for bringing this up, will implement this later today and push a new version. Let me know in case the new version doesn't work as expected, may take a few days for it to ship on all browsers.

**Comment by MerkaST** [comment]
This is a neat feature, thank you for implementing it.
I have however noticed the dates don't always seem to check out. I visited a channel page with videos I'd watched in the past, which showed the current date. In theory the extension should know them as I synchronised YouTube with the extension when I first installed it. Does it apply the current date if it sees a watched video it doesn't know about yet?
